bundles/org.openhab.binding.surepetcare/src/main/java/org/openhab/binding/surepetcare/internal/dto/SurePetcareBaseObject.java

```diff
@@ -32,7 +33,7 @@
     public ZonedDateTime updatedAt = ZonedDateTime.now();
 
     public Map<String, String> getThingProperties() {
-        Map<String, String> properties = new HashMap<String, String>();
+        Map<String, String> properties = new HashMap<@NonNull String, @NonNull String>();
```

Comment:
You already have `@NonNullByDefault` annotated at the top the class. So you shouldn't need to use `@NonNull` anywhere in this file.

Reply:
probably a left over from before I annoted the class. I've removed the NonNull annotations now.

Reply:
done